hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -7,21 +7,29 @@ import { RouteComponentProps } from "react-router-dom";
 import Header from "components/general/Header/Header";
 import { Grid } from "@material-ui/core";
 import Typography from "@material-ui/core/Typography";
+import { useDispatch } from "react-redux";
+import { getTeamInfoData } from "slices/event/teamDetailSlice";
 
 export interface PageParams {
     id: string;
 }
 
 const TeamDetail = ({ match }: RouteComponentProps<PageParams>) => {
+    // TODO: change api to use team_code instead of team_id
```

Comment:
i'm having trouble using the error selector. this is what my code looks like: 
![Screen Shot 2022-07-31 at 7 25 53 PM](https://user-images.githubusercontent.com/36282235/182049601-aa0f7c34-4384-4a2c-aa4b-ea24b1fc7b93.png)
when i try to load team info for an invalid team id, like abc, the error is briefly displayed, and then the page just repeatedly refreshes. i think it is because the combination of the dispatch and useSelector calls force the page to reload, i guess because the state changes, and then the page gets stuck in a cycle. i don't know how to prevent this constant reloading from happening.

Reply:
I think I need to see more of your render logic, we'll set up a call for this!